src/main/kotlin/racing/model/RacingGame.kt

```diff
@@ -0,0 +1,27 @@
+package racing.model
+
+import racing.condition.RacingCondition
+import racing.view.result.CarResultView
+
+class RacingGame(carCount: Int, private val round: Int, private val conditions: List<RacingCondition>) {
+    private val cars: List<Car>
+
+    init {
+        val cars = mutableListOf<Car>()
+        for (i in 1..carCount) cars.add(Car())
+        this.cars = cars
+    }
```

Comment:
코틀린의 확장함수 체이닝을 활용해보세요 :)
아래 처럼 IntRange, map 등을 활용해보시면 좋습니다 🙂
```suggestion
    private val cars: List<Car> = (1..carCount).map { Car() }
```

혹은 List 메서드를 활용해보셔도 좋겠어요 :)
아래 List는 생성자가 아닌 함수인데요, 이러한 형태를 Fake Constructor라고 부릅니다 :)
```suggestion
    private val cars: List<Car> = List(carCount) { Car() }
```

Reply:
훨씬 깔끔해지네요 감사합니다 👍